chef_master/source/chef_license_accept.rst

```diff
@@ -213,6 +217,19 @@ Specify the following argument:
 
   knife bootstrap --bootstrap-version 14.12.3
 
+``chef-run``
+-----------------------------------------------------
+``chef-run`` in Chef Workstation >= 0.3 has been updated to add support for accepting the license locally when remotely running Chef Infra Client 15.
+As of Chef Workstation <= 0.4 there is no way to manage the version of Chef Infra Client installed on the remote node.
+It defaults to the latest stable version available.
+
+To accept the license pass the command line flag, set the environment variable or add the following to your ``~/.chef-workstation/config.toml``:
```

Comment:
The first part of this sentence does not make sense... 
"To accept the license pass the command line flag,"

Reply:
There are 3 things they can do to accept the license:
1. Pass the `--chef-license` command line flag
2. Set the `CHEF_LICENSE` environment variable
3. Modify their `~/.chef/workstation/config.toml`

What is a better way for me to say that?

Reply:
Gotcha! I wasn't parsing that sentence correctly at all, especially when it came to inserting a comma. How about...: 
```suggestion
To accept the license, complete one of the following three tasks. Either pass the ``--chef-license`` command line flag, set the ``CHEF_LICENSE`` environment variable, or add the following to your ``~/.chef-workstation/config.toml`` file:
```